controller/RestController.php

```diff
@@ -3,7 +3,7 @@
 /**
  * RestController is responsible for handling all the requests directed to the /rest address.
  */
-class RestController extends Controller
+class RestController extends WebController
```

Comment:
Why this change? If there is something in WebController that RestController needs, it can be moved to the Controller base class, but I wouldn't make one inherit the other, they are intended to be sibling classes with different responsibilities.

Reply:
This was just to use the `WebController::linkUrlFilter` method. Moved `linkUrlFilter` to `Controller`. And will keep in mind the separation of concerns for both controllers and try to keep them well separated :)